astropy/cosmology/io/tests/test_table.py

```diff
@@ -206,6 +206,23 @@ def test_tofrom_table_mutlirow(self, cosmo_cls, cosmo, from_format, add_index):
         with pytest.raises(ValueError, match="more than one"):
             from_format(tbls, index=cosmo.name, format="astropy.table")
 
+    def test_tofrom_table_rename(self, cosmo, to_format, from_format):
```

Comment:
This test is very similar to `test_tofrom_row_rename()`, so it might be a good idea to write a single parametrized test, but that wouldn't work because the tests are in different files. Maybe it would be a good idea to organize the test differently, but even if it is then that would be beyond the scope of this pull request.